content/docs/manuals/networking-internals/semantic-addressing.md

```diff
@@ -1,12 +1,94 @@
 ---
-title: "Semantic Addressing"
+title: "Load Balancing"
 summary: ""
 draft: false
-weight: 311
+weight: 314
 toc: true
 seo:
   title: "" # custom title (optional)
   description: "" # custom description (recommended)
   canonical: "" # custom canonical URL (optional)
   noindex: false # false (default) or true
----
\ No newline at end of file
+---
+
+The load balancing in Oakestra consists of a multistep address conversion protocol using semantic addresses as part
+of an overlay network.
+
+### Semantic Addressing
+
+The semantic addressing happens on the service layer.
+Similar to a cluster IP in Kubernetes, these addresses reference all the instances (replicas) of a microservice
+with a single address. This address does not change when scaling up the instances or when migrating them.
+Unlike the Kubernetes cluster IP, when deploying a service in Oakestra, the platform provides as many Service IP
+addresses as the number of balancing policies supported (and active) within the platform.
+
+#### Example
+
+![Network Example - Dual Stack](_net_example_dual_stack.png)
+
+A developer can communicate with any instance of Service B either with **Round Robin** balancing policy or
+the **Closest** balancing policy. The former balances the traffic evenly between all the instances, and the
+latter finds the geographically closer instance.
+
+In the following tabs we go into further detail of what is happening with each above shown request.
+The ordering goes from top (First) to bottom (Fifth).
+
+{{< tabs "Requests" >}}
```

Comment:
What if in addition to FIrst etc we also add the envelope icon?
<img width="668" alt="image" src="https://github.com/user-attachments/assets/c70bd9a9-a862-4c63-a283-b0a9c612dd69">

I just pushed a new shortcode that allows you to provide an icon together with the tab name, it's called tabs-icon.

```suggestion
{{< tabs-icon "Requests" >}}
```

But I'm missing the icon of the last envelope, can you add it together with the .drawio file you used?

Reply:
I was also thinking of this, but didn't delve into how to realise it, so thanks for the shortcode! Will definitely use this.